src/sinks/util/metadata.rs

```diff
@@ -21,21 +20,19 @@ pub struct RequestMetadataBuilder {
 impl RequestMetadataBuilder {
     pub fn from_events<E>(events: &[E]) -> Self
     where
-        E: ByteSizeOf + EventCount + GetEventCountTags + EstimatedJsonEncodedSizeOf,
```

Comment:
This bound was entirely extraneous, because we already know exactly how many events we have, and in the case of this PR, I'm generating the request metadata for a bunch of `Metric`s, which doesn't have `EventCount` implemented... but `Event` and `TraceEvent` do, and the count is always 1, because, well, duh... so yeah, this was extraneous.